sdk/core/AzureCore/Source/Pipeline/Policies/ContentDecodePolicy.swift

```diff
@@ -194,6 +187,7 @@ public class ContentDecodePolicy: PipelineStage {
 
         // Store the logger so that the XML parser delegate functions can access it
         logger = response.logger
```

Comment:
The logger is a per-request object, so we should probably avoid storing it in a property. I would probably remove the property and then change this to:
```suggestion
        let logger = response.logger
```

Reply:
Fixed.